### ISSUE TITLE
Error importing config module

### ISSUE THREAD

**Issue by mohansn** [paulshannon/python-goodreads#1]
Following http://python-goodreads.readthedocs.org/en/latest/usage.html, I attempted this:
$ python
Python 2.7.3 (default, Aug  1 2012, 05:14:39) 
[GCC 4.6.3] on linux2
Type "help", "copyright", "credits" or "license" for more information.

> > > from config import DEVELOPER_KEY, DEVELOPER_SECRET
> > > Traceback (most recent call last):
> > >   File "<stdin>", line 1, in <module>
> > > ImportError: No module named config

There is no dependency mentioned in the docs. What module do I need to install?

**Comment by paulshannon** [comment]
The api requires a key and secret from Goodreads. I've updated the [usage docs](http://python-goodreads.readthedocs.org/en/latest/usage.html) to reflect this. The example provided is not a copy and paste solution.

**Comment by mohansn** [comment]
I still find it confusing. Someone looking to use this code will not be encouraged by the fact that the example code gives an unexplained ImportError.

Developers looking for a python wrapper to the Goodreads API likely already know that they need API keys.

**Comment by paulshannon** [comment]
I agree. I'm working on a better way. Along with some examples that work.

**Comment by mohansn** [comment]
Thank you, Paul. I appreciate it.
